snippets/strings.yml

```diff
@@ -0,0 +1 @@
+bytes_previous_versions_link: All previously released versions of this data are available at  <a href="https://www.nyc.gov/site/planning/data-maps/open-data/bytes-archive.page?sorts%5Byear%5D=0">BYTES of the BIG APPLE- Archive</a>-
```

Comment:
Extraneous dash at the end of this?

Reply:
yes, thanks for fixing. Luckily this hasn't be actually deployed anywhere, though I think I would have caught it in review. But now that things have settled, it's time to stop pushing code without reviewers. (The real review in these cases has been with GIS, looking at the finished product on Socrata)